package.json

```diff
@@ -76,7 +77,7 @@
     "ember-drag-drop": "^0.9.0-beta.0",
     "ember-feature-flags": "^6.0.0",
     "ember-fetch": "^8.1.2",
-    "ember-intl": "^5.7.2",
+    "ember-intl": "^6.1.0",
```

Comment:
There is an issue with the `ember-intl` 6.x packages at the moment. We should probably remain on 5.7.2 for now.

Reply:
This was a confusion from the different version on the `feat/rdfa-links-ui` branch of the editor repo. I've corrected that now and I'll remove the commit from here.